app/models/injection_attempt.rb

```diff
@@ -29,4 +29,10 @@ def error_messages
     messages = data&.fetch(:errors)&.map { |child| child[:error] }
     messages || []
   end
+
+  private
+
+  def ignorable?
```

Comment:
do you want this in this PR?

Reply:
No... that was _supposed_ to have been committed on a different branch 🤦‍♂ 

will force push a replacement

Reply:
👍